src/java/voldemort/server/VoldemortServer.java

```diff
@@ -1,12 +1,12 @@
 /*
  * Copyright 2008-2013 LinkedIn, Inc
- * 
```

Comment:
Could I keep this change since I was using Felix's formatting rule?
The unnecessary white spaces at the end of line should be removed.